### DISCUSSION TITLE
Styling questions from v0.5 to v1.4

### DISCUSSION THREAD

**Discussion by toastedqu** [thuliteio/doks#1221] [Q&A] [answered]
Hi, I just upgraded my web from v0.5 to v1.4, and I have encountered the following issues. How do I fold the left sidebar menus on default instead of listing all sections?

**Comment by toastedqu** [comment]
May I get an answer for this?

**Comment by h-enk** [comment, marked as answer]
That's not an option you can currently set (interesting though and fairly easy to implement).

For now, copy `node_modules/@hyas/doks-core/layouts/partials/sidebar/render-section-menu.html` to `/layouts/partials/sidebar/render-section-menu.html` and update line 89 to look like this:

```html
    <details{{- with $ariaCurrent }} open{{- end }}>
```

**Reply by h-enk** [reply]
https://github.com/gethyas/doks/issues/1227

**Reply by toastedqu** [reply]
Thanks it worked!
